CONTRIBUTING.md

```diff
@@ -70,21 +70,21 @@ move forward. See editor Lee Byron talk about
 * **Enable new capabilities motivated by real use cases**
 
   Every change should intend on unlocking a real and reasonable use case. Real
-  examples are always more interesting than theoretical ones, and common
-  scenarios are more interesting than rare ones. RFCs should do more than offer
+  examples are always more compelling than theoretical ones, and common
+  scenarios are more compelling than rare ones. RFCs should do more than offer
   a different way to reach an already achievable outcome.
 
 * **Simplicity and consistency over expressiveness and terseness**
 
-  There are plenty of behaviors and patterns found in other languages
+  There are plenty of behaviors and patterns found in other languages that are
   intentionally absent from GraphQL. "Possible but awkward" is often favored
   over more complex alternatives. Simplicity (e.g. fewer concepts) is
   more important than expressing more sophisticated ideas or writing less.
 
 * **Preserve option value**
 
-  It's hard to know what the future brings, so whenever possible decisions
-  should be made which allow for more options in the future. Sometimes this is
+  It's hard to know what the future brings, so whenever possible, decisions
+  should be made that allow for more options in the future. Sometimes this is
```

Comment:
This comma isn't quite right - it creates a parenthesised text which cannot be safely deleted. Effectively you want the "whenever possible" to be parenthesized, as in:

```
  It's hard to know what the future brings so, whenever possible, decisions
  should be made that allow for more options in the future. Sometimes this is
```

Personally, I'd rewrite it as:

```suggestion
  It's hard to know what the future brings; whenever possible, decisions
  should be made that allow for more options in the future. Sometimes this is
```